src/agent/agent_info/include/agent_info.hpp

```diff
@@ -11,13 +12,16 @@ class AgentInfo
     std::string GetName() const;
     std::string GetKey() const;
     std::string GetUUID() const;
+    std::vector<std::string> GetGroups() const;
 
     void SetName(const std::string& name);
     void SetKey(const std::string& key);
     void SetUUID(const std::string& uuid);
+    void SetGroups(const std::vector<std::string>& group_list);
```

Comment:
Prefer camelCase for function parameters.

Reply:
Done!